src/Container/AddWmsPanel/AddWmsLayerEntry/AddWmsLayerEntry.tsx

```diff
@@ -96,15 +96,25 @@ export class AddWmsLayerEntry extends React.Component<AddWmsLayerEntryProps, Add
         <div className="add-wms-layer-entry">
           {layerTextSpan}
           {
-            copyright
-              ? <Icon className="add-wms-add-info-icon" name="copyright" aria-label="attribution-info" />
+            copyright ? (
+              <FontAwesomeIcon
+                className="add-wms-add-info-icon attribution-info"
+                icon={['far', 'copyright']}
+                data-testid="attribution-info"
```

Comment:
Is there really no other way to get the component in the test?

Reply:
you mean, by using another property? Well, yes. I could try to find it by role `img` and check for images on screen or I can write a custom query for a specific property but the end will be the same.
Is the property name that bothers why? I think using `data-testid` says exactly the purpose of using this property and there is a method already implemented in Testing Library for this.

Reply:
I habe read that elsewhere also. Will this be visible for users somehow, does it end up in the dom?

Reply:
Yes, it would have been visible as far as I know. But there is a new commit that removes it